psa-crypto-sys/build.rs

```diff
@@ -52,30 +57,22 @@ fn setup_mbed_crypto() -> Result<()> {
     }
 
     // Build the MbedTLS libraries
-    if !::std::process::Command::new("make")
-        .arg("-C")
-        .arg(&mbedtls_dir)
-        .status()
-        .or_else(|_| Err(Error::new(ErrorKind::Other, "making mbedtls failed")))?
-        .success()
-    {
-        return Err(Error::new(
-            ErrorKind::Other,
-            "making mbedtls returned an error status",
-        ));
-    }
+    let mbed_build_path = Config::new("vendor")
```

Comment:
The `cmake` crate automatically builds in `OUT_DIR`?

Reply:
Yes, that's one of the main reasons why I went with the crate version :)